.github/workflows/test.yml

```diff
@@ -53,10 +53,10 @@ jobs:
     - name: Setup node
       uses: actions/setup-node@v2
       with:
-        node-version: '14'
+        node-version: '16'
```

Comment:
should this match what's in the .nvmrc ?

Reply:
https://github.com/sanger/labwhere/blob/develop/.github/workflows/automated_release_and_build.yml as an example.

Reply:
Huh, thought I'd already done that, but also thought I'd updated the basic node version to 16, so obviously got a bit mixed up somewhere. Will do. (And try 16 as well)